static/elements/chromedash-featurelist.js

```diff
@@ -388,24 +383,26 @@ class ChromedashFeaturelist extends LitElement {
   }
 
   render() {
-    const filteredWithState = this.filtered.map((feature) => {
+    console.log('num features = ' + this.filtered.length);
+    let filteredWithState = this.filtered.map((feature) => {
       return {
         feature: feature,
         open: this.openFeatures.has(feature.id),
         subscribed: this.subscribedFeatures.has(String(feature.id)),
       };
     });
+    let numOverLimit = 0;
+    if (filteredWithState.length > MAX_FEATURES_SHOWN) {
+      numOverLimit = filteredWithState.length - MAX_FEATURES_SHOWN;
+      filteredWithState = filteredWithState.slice(0, MAX_FEATURES_SHOWN);
+    }
     return html`
       <link rel="stylesheet" href="/static/css/elements/chromedash-featurelist.css">
       <style>
         .item {width: 100%}
       </style>
 
-      <lit-virtualizer
-        .scrollTarget=${window}
-        .items=${filteredWithState}
-        @rangechange=${this._onScrollList}
```

Comment:
Do we want to recreated this feature? - highlight the left sidebar item when scrolling.

(If not, remove _onScrollList in this component.)

Reply:
Let's try removing it and see if anyone complains.